apps/health_campaign_field_worker_app/lib/utils/constants.dart

```diff
@@ -1,8 +1,31 @@
+import 'package:collection/collection.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../blocs/app_initialization/app_initialization.dart';
 
 class Constants {
   static const String baseURL = 'https://health-dev.digit.org/';
-  static const String mdmsApiEndPoint = 'egov-mdms-service/v1/_search';
+  static const String mdmsApiPath = 'egov-mdms-service/v1/_search';
```

Comment:
read this from service registry and default to this constant if not found.

Reply:
@egov-joy  This is utilised to call the service registry.

Reply:
yup, agreed this is a little bit of a chicken and egg thing but trying to see if there is anyway we can support the MDMS url also getting changed.. so that for subsequent calls after service registry is downloaded we can use the updated endpoint rather than the hard coded one.
If this is too much of a hassle and not a realistic scenario, we can drop this for now.

Reply:
@egov-joy , @ajilo-eGov  can we set this in env so that this can be changes at later point. if required